docs/ops/generation/EyeLike_9.md

```diff
@@ -0,0 +1,142 @@
+## EyeLike <a name="EyeLike"></a> {#openvino_docs_ops_generation_EyeLike_9}
+
+**Versioned name**: *EyeLike-9*
+
+**Category**: *Generation*
+
+**Short description**: *EyeLike* operation generates shift matrix.
+
+**Detailed description**:
+
```

Comment:
I think we need to generate a batch of identity matrices as well as TensorFlow: https://www.tensorflow.org/api_docs/python/tf/eye

Reply:
Done, thanks!